### PR TITLE
dev-vcs/tortoisehg: update live version

### PR DESCRIPTION
**Pull request by winterheart**
Update HG_URI_REPO and SRC_URI, fix keywords, misc fixes.
Update to new 5.5 version, remove old version.
Closes: https://bugs.gentoo.org/737902

### CODE REVIEW

File: dev-vcs/tortoisehg/tortoisehg-5.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	distutils-r1_python_install_all
+	dodoc doc/ReadMe*.txt doc/TODO contrib/mergetools.rc
+	newicon -s scalable icons/scalable/apps/thg.svg thg_logo.svg
+	domenu contrib/thg.desktop
+}
+
+pkg_postinst() {
+	elog "When startup of ${PN} fails with an API version mismatch error"
+	elog "between dev-python/sip and dev-python/PyQt5 please rebuild"
+	elog "dev-python/qscintilla-python."
```
Comment (juippis) [inline dev-vcs/tortoisehg/tortoisehg-5.5.ebuild]: This sounds like something that could be solved via subslots in the ebuild, but it's not your task to do in here. Just weird no one else did it already.

File: dev-vcs/tortoisehg/tortoisehg-5.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit desktop distutils-r1
```
Comment (juippis) [inline dev-vcs/tortoisehg/tortoisehg-5.5.ebuild]:
```
 * QA Notice: new icons were found installed but icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/scalable/apps/thg_logo.svg
 * Please make sure to call xdg_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
```

Same for live version.

File: dev-vcs/tortoisehg/tortoisehg-5.5.ebuild (hunk 1 above)
Comment (winterheart) [reply dev-vcs/tortoisehg/tortoisehg-5.5.ebuild]: added dev-python/PyQt5:= subslot check.

**Comment by winterheart** [comment]
@juippis updated ebuilds, version bumped to 5.5.1.

File: dev-vcs/tortoisehg/tortoisehg-5.5.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+
+RDEPEND="
+	${HG_DEPEND}
+	dev-python/iniparse[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/PyQt5:=[network,svg,${PYTHON_USEDEP}]
+	>=dev-python/qscintilla-python-2.9.4:=[qt5(+),${PYTHON_USEDEP}]"
```
Comment (juippis) [inline dev-vcs/tortoisehg/tortoisehg-5.5.1.ebuild]: These packages don't have subslots, so `:=` are not working here. Rest looks good, I'll remove them.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**